control-plane/.gitignore

```diff
@@ -0,0 +1,14 @@
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+firebase-debug.log*
+firebase-debug.*.log*
+
+# Firebase cache
+.firebase/
+
```

Comment:
Please add `.firebaserc`
I recommend to copy the entire .gitignore from https://github.com/GoogleChromeLabs/trust-safety-demo/blob/main/conversion-measurement/.gitignore including the comments and uncomment the line with `.firebaserc`